src/assets/js/protocol/protocol-footer.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var _mqWide = matchMedia('(max-width: 479px)');
+
+    if (_mqWide.matches) {
+        window.Mozilla.Details.init('.mzp-c-footer-sections .mzp-c-footer-heading');
```

Comment:
Should we also add a check that `Mozilla.Details` is defined before initializing?

Reply:
Added a check 👍